plivo.py

```diff
@@ -20,15 +22,28 @@ class PlivoError(Exception):
     pass
 
 
-def validate_signature(uri, post_params, signature, auth_token):
-    for k, v in sorted(post_params.items()):
+def validate_signature(uri, params, signature, auth_token):
+    qs = urlparse(uri).query
+    if qs:
+        params_from_qs = dict(parse_qsl(qs))
```

Comment:
https://www.plivo.com/docs/xml/request/#validation doesn't mention anything about combining query string params with form data. Is that really how it's supposed to be done? And if you *really* need to consider query string params in the signature, what happens if I e.g. set my Message URL to `https://example.com/message?somekey=someval`? Does Plivo ever append their own query string params to a URL?